app/test/v2/test_meetups.py

```diff
@@ -0,0 +1,47 @@
+"""test for api endpoints"""
+# inbuilt modelues
```

Comment:
Minor typo: `modules`

Reply:
NOTED THAT. i will change on this in my next PR.